open-api/rest-catalog-open-api.yaml

```diff
@@ -1630,6 +1990,102 @@ components:
         metadata-log:
           $ref: '#/components/schemas/MetadataLog'
 
+    SQLViewRepresentation:
+      type: object
+      required:
+        - type
+        - sql
+        - dialect
+      properties:
+        type:
+          type: string
+        sql:
+          type: string
+        dialect:
+          type: string
+
+    ViewRepresentation:
+      oneOf:
+        - $ref: '#/components/schemas/SQLViewRepresentation'
+
+    ViewHistoryEntry:
+      type: object
+      required:
+        - version-id
+        - timestamp-ms
+      properties:
+        version-id:
+          type: integer
+        timestamp-ms:
+          type: integer
+          format: int64
+
+    ViewVersion:
+      type: object
+      required:
+        - version-id
+        - timestamp-ms
+        - schema-id
+        - summary
+        - representations
+        - default-namespace
+      properties:
+        version-id:
+          type: integer
+        timestamp-ms:
+          type: integer
+          format: int64
+        schema-id:
+          type: integer
```

Comment:
This can be -1 to signal that it should be the last added schema?

Reply:
we don't send a -1 for schema-id because we don't provide a way to set the current schema id like we do for table metadata

Reply:
@nastra, we need to replace the schema ID and pass -1 here if the schema was just added. The server side may reassign the schema ID so it needs to be replaced with a placeholder.

Reply:
this should be done now with the last commit